packages/apps/src/configs/storages/evm-storage.ts

```diff
@@ -0,0 +1,93 @@
+import { EvmChainStorage } from '@webb-dapp/apps/configs/storages/evm-chain-storage.interface';
+import { StorageHandler } from '@webb-dapp/utils';
+import { EVMStorage } from '@webb-dapp/react-environment/api-providers/web3';
+
+const rinkebyStore = new EvmChainStorage([
+  {
+    size: 0.1,
+    address: '0x876eCe69618e8E8dd743250B036785813824D2D7',
+    symbol: 'ETH',
+    syncedBlock: 120000, // should be hardcoded to deployed block number
```

Comment:
I still don't know what syncedBlock means. This should err on the side of being as verbose as needed. Is this the latest synced block? If so, lets call it, `latestSyncedBlock` or `lastSyncedBlock` or `lastQueriedBlock`, etc.

I think we should also add another parameter for `createdAtBlock` to indicate the block at which these mixers were created.  This way, for new clients, we can begin fetching events immediately from the point they were created rather than before ever, mistakenly

Reply:
I agree the field can be named something more descriptive like `lastQueriedBlock`.

I think that these `EvmChainStorage` objects will be passed default values as params to the constructor. Therefore, the `lastQueriedBlock` can pass the deployed block number, and new clients will only ever start querying from the deployed block.  In the constructor, we can read from the localStorage to determine if the client has anything in storage for the particular mixer.